Withdrawal logic completed

Cristos123 opened the pull request:
resolved #6 
- The withdrawal for user and admin page to manage the withdrawal have been completed

Cristos123 committed:
Withdrawal logic completed

Cristos123 committed:
Withdrawal Updated

josephajibodu commented on app/Http/Controllers/WithdrawalController.php:
```diff
@@ -0,0 +1,134 @@
+    {
+        // $withdrawals = auth()->user()->withdrawals;
+        // $withdrawals = Withdrawal::find(auth()->user()->id);
+        $withdrawals = Withdrawal::where('user_id', auth()->user()->id)->get();
```
Paginate this. 

```php
$withdrawals = Withdrawal::where('user_id', auth()->user()->id)->paginate(25);
```

josephajibodu commented on app/Models/User.php:
```diff
@@ -63,4 +64,14 @@ public function kyc(): HasOne
+     *
+     * @return \Illuminate\Database\Eloquent\Relations\HasOne
+     */
+    public function withdrawals(): HasOne
```
Change the relationship to hasMany

josephajibodu commented on app/Models/Asset.php:
```diff
@@ -15,4 +15,13 @@ public function investment(): BelongsTo
+     *
+     * @return \Illuminate\Database\Eloquent\Relations\HasOne
+     */
+    public function withdrawal(): HasOne
```
Change this relationship to hasMany.

Many withdrawals can belong to an Asset. So its a one-to-many  relationship.

Cristos123 committed:
Fixing issues on users model and asset models
from one to one relationship to hasMany in both models
update index function on withdrawal controller
